3rd cycle

chukwuebukaVictor opened the pull request:
This project displays a responsive mobile and desktop version of my portfolio page.

chukwuebukaVictor committed:
Add style to nav bar

Add some elements in html file
Add styles to nave bar
Add some images
Modify my README.md file

chukwuebukaVictor committed:
Add style to my first section

chukwuebukaVictor committed:
Add style to body section for desktop version

chukwuebukaVictor committed:
Add elements to css and html file

Add Rubby on rails work list tom my html
Add styles to my last section details for desktop version

chukwuebukaVictor committed:
Add style to my footer

Add styles to my contact elements
Add background image to my contact elements

chukwuebukaVictor committed:
Add screen shot image to README.md file

chukwuebukaVictor committed:
Add screen shot image to README.md file

SajjadAhmad14 commented on styles.css:
```diff
@@ -369,3 +371,188 @@ form {
+  display: none;
+}
+
+@media (min-width: 768px) {
```
Please use the value for min-width property in range of 992px - 1100px.

SajjadAhmad14 commented on styles.css:
```diff
@@ -369,3 +371,188 @@ form {
+    background-color: #ebebff;
+  }
+
+  .about-myself,
```
Please use the same font color for heading, languages, frameworks, and skills as in the design.

SajjadAhmad14 commented on styles.css:
```diff
@@ -369,3 +371,188 @@ form {
+    display: none;
+  }
+
+  .intro h1 {
```
Please use the same color as defined in the design.

SajjadAhmad14 commented on index.html:
```diff
@@ -35,143 +40,173 @@ <h1>I’m Victor Ozoede Glad to see you!</h1>
-            <li>html</li>
-            <li>css</li>
-            <li>javaScript</li>
+    <section class="work-three work">
```
Font colors for project headings in the work section are also not the same.

SajjadAhmad14 commented on index.html:
```diff
@@ -35,143 +40,173 @@ <h1>I’m Victor Ozoede Glad to see you!</h1>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <ul class="work-bottom2">
```
The height of the buttons is a little bit large on the tablet view. Please see the design and adjust the height. Kindly see the below screenshot for reference.
![buttons](https://user-images.githubusercontent.com/35504149/130109652-5818138e-4f39-40d4-8c92-25c888d75f37.png)

SajjadAhmad14 reviewed (changes requested):
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Victor Ozoede, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
- [ ] GitHub flow is used properly.
- [ ] README is professional.
## To Improve 🙏

- [ ] Please check inline comments under the review
- [ ] Please adjust buttons height on the medium viewport, and follow font colors from design. 
## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

chukwuebukaVictor committed:
Format program list elements

Add another color to the program list
Change media querry and format program list tablet live demo

chukwuebukaVictor committed:
Modify footer

Change the position of the footer to align at the center

chukwuebukaVictor committed:
Fix linter error

Emmyn5600 commented on index.html:
```diff
@@ -35,143 +40,173 @@ <h1>I’m Victor Ozoede Glad to see you!</h1>
-          <li><img src="./img/html-logo.svg" alt="html-logo"></li>
-          <li>html</li>
+      <div class="flex-item1 flex-item">
+        <h1 class="myself-header">about myself</h1>
+        <p class="myself-paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me. </p>
```
- [ ] Optional suggestions

- Work on the `letter-spacing` in your paragraph to get same we have in the figma.

- You can also change the `JavaScript HTML CSS ` seem to be broken.

Emmyn5600 reviewed (approved):
Hi, 
Good job so far! 👏🏼 @chukwuebukaVictor 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed, however use a descriptive feature branch:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
